### DISCUSSION TITLE
High error in energy

### DISCUSSION THREAD

**Discussion by baoanh13** [mir-group/allegro#86] [Q&A]
Hello,

I am trying to train a model of a system that contains > 4000 training data with 3 type of elements: Fe, C, H, by Nequip and Allegro.    The error in training session by nequip seems good but the training time is rather slow, so I tried Allegro to accelerate, but it seems the error in training session by Allegro is not good, especially in energy. 

This is the latest results after 2 days for Nequip:

<img width="1852" alt="image" src="https://github.com/mir-group/allegro/assets/131165028/4470d13f-f5d9-46f8-9119-7c24b599e4ff">

And this is by Allegro after 3 days:

<img width="1851" alt="image" src="https://github.com/mir-group/allegro/assets/131165028/9d11cf0f-7aae-4b90-9ffa-1d4f93be055b">

In the train.yaml file, in both case I used the loss coeffs as the default
loss_coeffs:
  forces: 1.
  total_energy:
    - 1.
    - PerAtomMSELoss

I don't know what is the reason causes the bad performance in energy training by Allegro. Are there any better loss function values or another hyperparameters to improve the accuracy?

I attach the yaml file for training in both cases I use as below.

[train_allegro.txt](https://github.com/mir-group/allegro/files/15258445/train_allegro.txt)
[train_nequip.txt](https://github.com/mir-group/allegro/files/15258446/train_nequip.txt)

Thank you for your concern.
